pyro_models/arm/radon_complete_pool.py

```diff
@@ -35,7 +35,7 @@ def model(data, params):
     # initialize transformed parameters
     # model block
 
-    sigma =  pyro.sample("sigma", dist.Cauchy(0., 2.5))
+    sigma =  pyro.sample("sigma", dist.HalfCauchy(0., 2.5))
```

Comment:
`HalfCauchy` is centered at 0 so i think it should be `dist.HalfCauchy(2.5)`

Reply:
Ah yes! Let me change that